pkg/forwarder/interceptor/interceptor.go

```diff
@@ -107,6 +124,51 @@ func (i *interceptor) CreateContainer(ctx context.Context, req *pb.CreateContain
 	return res, err
 }
 
+func isTargetPath(path, targetPath string) bool {
+	return targetPath != "" && targetPath == path
+}
+
+func waitForDeviceMounted(path string) error {
+	timer := time.NewTimer(volumeTimeout)
```

Comment:
We are using the `retry` package to simplify this kind of retry logic in other places.

https://github.com/confidential-containers/cloud-api-adaptor/blob/a771f96274c8b98f9f450b9477ad0572902ae94e/pkg/adaptor/proxy/proxy.go#L121-L130